Procfile

```diff
@@ -0,0 +1 @@
+web: bin/boot
```

Comment:
I'm not super sure, I recall that we added this while getting it to deploy to Heroku because we came across this issue from the README: https://elements.heroku.com/buildpacks/heroku/heroku-buildpack-static (under Procfile / multiple buildpacks).

I think perhaps Heroku automatically picked up on nanoc's presence previously. Could this cause an issue with the current build process?